Demo0.java

```diff
@@ -1,9 +1,15 @@
 // Prints the numbers 0 to 5.
 // (Same program as shown in lecture 1-1).
-public class PrintTo5 {
+public class Demo0 {
```

Comment:
No need to change the class name to the one you saw in class/recitation
Keep it PrintTo5 :)